tests/functional/member_checks.py

```diff
@@ -207,3 +207,15 @@ class Cls(enum.IntEnum):
 
 
 SOME_VALUE = Cls.Baz  # [no-member]
+
+
+from contextlib import contextmanager
+
+
+@contextmanager
+def context_manager_to_check_members():
+    yield
+
+cm = context_manager_to_check_members() # pylint: disable=invalid-name
```

Comment:
You can rename `cm` to `context_manager` and remove the invalid-name.

Reply:
well, the message was `Constant name "cm" doesn't conform to UPPER_CASE naming style (invalid-name)`
so I had to do it differently.